explainability-core/src/main/java/org/kie/trustyai/explainability/model/Dataframe.java

```diff
@@ -1208,6 +1209,51 @@ public String toString() {
         return builder.toString();
     }
 
+    public Dataframe std() {
+
+        // public Dataframe copy() {
+        // return new Dataframe(
+        // this.data.stream().map(ArrayList::new).collect(Collectors.toCollection(ArrayList::new)),
+        // metadata.copy(), internalData.copy());
+        // }
+
+        // Dataframe(List<List<Value>> data, Metadata metadata) {
+        // this.data = new ArrayList<>(data);
+        // this.metadata = metadata;
+        // this.internalData = new InternalData();
+        // }
+
+        final int numCols = metadata.names.size();
+
+        // Dataframe assumes this is list of columns each containing a list of rows
+        final List<List<Value>> stdData = new ArrayList<>(numCols);
+
+        final StandardDeviation sd = new StandardDeviation();
+
+        for (int i = 0; i < numCols; i++) {
```

Comment:
Can you only iterate through numeric columns, skipping non-numerics? See [here](https://github.com/trustyai-explainability/trustyai-explainability/blob/4dabe5409efd3ead002d83b00d154f3c634b3db9/explainability-core/src/main/java/org/kie/trustyai/metrics/drift/meanshift/Meanshift.java#L35) as an example of how to do this

Reply:
I drop the non-numeric columns now.